llvm/lib/Transforms/Vectorize/VPlanTransforms.h

```diff
@@ -104,7 +104,8 @@ struct VPlanTransforms {
   /// VPCanonicalIVPHIRecipe with a VPEVLBasedIVPHIRecipe.
   /// VPCanonicalIVPHIRecipe is only used to control the loop after
   /// this transformation.
-  static void addExplicitVectorLength(VPlan &Plan);
+  /// \returns true if the transformation succeeds, or false if it doesn't.
+  static bool addExplicitVectorLength(VPlan &Plan);
```

Comment:
nit: `tryAddExplicitVectorLength` would be more accurate now?

Reply:
Fixed. Thanks!